app/assets/javascripts/application.js.erb

```diff
@@ -191,15 +191,6 @@ var HelperModule = (function(){
 
   var helpers = {};
 
-  helpers.treeLinkTruncation = function() {
```

Comment:
We still call this function in `app/assets/javascripts/application.js.erb` line 248.